### DISCUSSION TITLE
AAD Connect

### DISCUSSION THREAD

**Discussion by KlekotJ** [AzureCommunityPL/group#174] [Pomoc] [answered]
Wszędzie w internecie piszą że hasła synchronizują się automatycznie co 2 minuty. A u mnie co 30.  Gdzie to można zmienić?

**Comment by eklime** [comment, marked as answer]
Żeby ryby na talerzu nie podawać, bo to w dłuższej perspektywie nic Ci to nie da to...
1. "Wszędzie w internecie" - poprosimy linki do artykułów i wpisów w tym internecie gdzie o tym czytałeś.
2. Jaką metodą weryfikowałeś, że to faktycznie jest 30 a nie 2 minuty?
3. Czy sam instalowałeś AAD Connect, czy zastałeś już po kimś.
4. W jakiej konfiguracji został zainstalowany?
5. Czy ktoś grzebał przy harmonogramach?

Czekamy na odpowiedzi ;)

**Reply by KlekotJ** [reply]
1. https://www.rebeladmin.com/2017/11/azure-ad-password-synchronization/
https://docs.microsoft.com/pl-pl/azure/active-directory/hybrid/how-to-connect-password-hash-synchronization
2. 
![image](https://user-images.githubusercontent.com/75643941/101456514-76d9cd00-3934-11eb-89ec-046d177bc99d.png)
3. Sam instalowałem lata temu i chyba zawsze tak było, ale dopiero teraz mi to przeszkadza.
4. Nie wiem co tutaj napisać. Zwykła domena, dwa kontrolery. Na jednym z serwerów jest AAD Connect, który łączy się do chmury. 
5. Harmonogram nie był nigdy zmieniany.

**Reply by smereczynski** [reply]
A zmieniałeś hasło i weryfikowałeś czy po max 2min masz je zmienione w AAD? :)

**Reply by PoborcaMaciej** [reply]
Spojrz moze na ten artykul i zobacz te logi po twojej stronie: 
https://www.thecloudtechnologist.com/azure-ad-connect-dirsync-password-sync-taking-too-long/

**Reply by KlekotJ** [reply]
Sprawdziłem na zasadzie, że zmieniam hasło i próbuję się logować na nowe i faktycznie wygląda na to, że działa co 2 minuty. To zamyka temat. Dziękuję za pomoc.

**Reply by eklime** [reply]
Git. To na przyszłość jeszcze jedna uwaga - to co wyświetla portal nijak nie może być traktowane jako wiarygodne i dokładne źródło informacji.
Podstawowym narzędziem pracy w tym wypadku powinien być PowerShell i sprawdzenie przez Get-AzureADUser atrybutu "PasswordLastSet" (tu z pamięci, ale coś w tych okolicach)
Dodatkowo masz usługę synchronizatora na serwerze - tam możesz sprawdzić kiedy były wykonywane synchronizacje i co było aktualizowane.

**Comment by mgrabarz** [comment]
@KlekotJ czy sugestia od @PoborcaMaciej  pomogła rozwiązać problem?
https://www.thecloudtechnologist.com/azure-ad-connect-dirsync-password-sync-taking-too-long/

**Reply by KlekotJ** [reply]
Tak. Pomogła. 
Dziwne, bo pisałem już odpowiedź na tego posta a teraz jej nie widzę. 
Zrobiliśmy próby z resetem hasła i logowaniem na nowe i to potwierdziło synchronizację co 2 minuty. Dziękuję za pomoc.

**Reply by mgrabarz** [reply]
Okazuje się, że rzeczywiście jest Twoja odpowiedź, tylko trzeba "expand" zrobić w wątku powyżej. 
Dzięki za potwierdzenie!
